src/Strider.c

```diff
@@ -0,0 +1,198 @@
+#include <pebble.h>
+
+static Window *s_window;
+static Layer *s_window_layer, *s_dots_layer, *s_progress_layer, *s_average_layer;
+static TextLayer *s_time_layer, *s_step_layer;
+
+static char s_current_time_buffer[8], s_current_steps_buffer[16];
+static int s_step_count = 0, s_step_goal = 0, s_step_average = 0;
+
+GColor ColorLoser;
+GColor ColorWinner;
+
+// Is health data available?
+bool is_health_available() {
+  return HealthServiceAccessibilityMaskAvailable &
+    health_service_metric_accessible(HealthMetricStepCount,
+      time_start_of_today(),
+      time(NULL));
+}
+
+// Daily step goal
+static void get_step_goal() {
+  s_step_goal = (int)health_service_sum_averaged(HealthMetricStepCount,
+    time_start_of_today(),
```

Comment:
I would break these parameter values into const variables, especially seeing as you're calling time_start_of_today() twice.